corehq/apps/export/utils.py

```diff
@@ -165,6 +165,10 @@ def convert_saved_export_to_export_instance(domain, saved_export, dryrun=False):
                     ))
 
                 new_column = _get_normal_column(new_table, column_path, transform)
+
+                # If we can't find the column in the current table
+                # look in every other table to see if the column is a repeat
+                # that did not receive more than one entry
```

Comment:
i think it's the other way around, we're looping over the old columns. if we find a column in the main table and it can't found, then it might be repeat question so we need to check all the new tables to see if it can be found there. we need to do that check for every table besides the main table for the case you mentioned, nested repeats